pkg/rancher-desktop/utils/shortcuts.ts

```diff
@@ -0,0 +1,207 @@
+import os from 'os';
+
+import { BrowserWindow } from 'electron';
+
+import { toArray } from '@pkg/utils/array';
+
+type Platform = 'darwin' | 'linux' | 'win32';
```

Comment:
Do you think it makes sense for this type to be `NodeJS.Platform`? That should match the return type of `os.platform()`.

Reply:
Definitely `NodeJS.Platform`